components/core/tests/test-EncodedVariableInterpreter.cpp

```diff
@@ -241,14 +241,25 @@ TEST_CASE("EncodedVariableInterpreter", "[EncodedVariableInterpreter]") {
 
         // Test encoding
         vector<encoded_variable_t> encoded_vars;
-        vector<string> var_strs = {"4938", to_string(EncodedVariableInterpreter::get_var_dict_id_range_begin()), "-25.5196868642755", "-00.00"};
+        vector<variable_dictionary_id_t> added_var_ids;
+        vector<string> var_strs = {"4938", to_string(EncodedVariableInterpreter::get_var_dict_id_range_begin()), "-25.5196868642755", "-00.00", "bin/python2.7.3"};
         msg = "here is a string with a small int " + var_strs[0] + " and a very large int " + var_strs[1] + " and a double " + var_strs[2] +
-              " and a weird double " + var_strs[3];
+              " and a weird double " + var_strs[3] + " and a str with numbers " + var_strs[4];
         LogTypeDictionaryEntry logtype_dict_entry;
-        EncodedVariableInterpreter::encode_and_add_to_dictionary(msg, logtype_dict_entry, var_dict_writer, encoded_vars);
-
+        EncodedVariableInterpreter::encode_and_add_to_dictionary(msg, logtype_dict_entry, var_dict_writer, encoded_vars, added_var_ids);
         var_dict_writer.close();
 
+        // Test added_var_ids is correctly populated
+        size_t encoded_var_id_count = 0;
+        for(const auto& var : encoded_vars) {
```

Comment:
Current convention throughout the code is to have a space after the `for` and `if` and before the `{`